CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Head
 
 - Fixed: bug causing rules in extended configs to be merged with, rather than replaced by, the extending config.
+- Fixed: `selector-max-specificity` now ignores selectors containing `:matches()` pseudo-classes, and throws a parse error if the underlying `specificity` module cannot parse the selector. 
```

Comment:
I think the point here is that it *does not* throw a parse error, right? Instead, it just logs a standard PostCSS warning?